homeassistant/components/velux/cover.py

```diff
@@ -118,3 +139,24 @@ async def async_set_cover_position(self, **kwargs):
     async def async_stop_cover(self, **kwargs):
         """Stop the cover."""
         await self.node.stop(wait_for_completion=False)
+
+    async def async_close_cover_tilt(self, **kwargs):
+        """Close cover tilt."""
+        await self.node.close_orientation(wait_for_completion=False)
+
+    async def async_open_cover_tilt(self, **kwargs):
+        """Open cover tilt."""
+        await self.node.open_orientation(wait_for_completion=False)
+
+    async def async_stop_cover_tilt(self, **kwargs):
+        """Stop cover tilt."""
+        await self.node.stop_orientation(wait_for_completion=False)
+
+    async def async_set_cover_tilt_position(self, **kwargs):
+        """Move cover tilt to a specific position."""
+        if ATTR_TILT_POSITION in kwargs:
```

Comment:
We don't need this check. This parameter is required.

Reply:
Addressed in https://github.com/home-assistant/core/pull/45596